lib/omnibus/builder.rb

```diff
@@ -762,12 +762,11 @@ def build
         end
       end
 
-      build_time = measure("Build #{software.name}") do
+      measure("Build #{software.name}", lambda { |d| software.store_build_duration(d) }) do
```

Comment:
I think this should work, too:

```suggestion
      measure("Build #{software.name}", software.method(:store_build_duration) do
```